.gitignore

```diff
@@ -17,8 +17,10 @@ out
 build
 dependencies.lock
 **/dependencies.lock
-gradle/
```

Comment:
Why are we removing the directory exclusion?

Reply:
Because we have legit content in `gradle/` (ignores for the relevant files are added)